nrf_security/Kconfig.legacy

```diff
@@ -232,6 +232,7 @@ if MBEDTLS_LEGACY_CRYPTO_C
 
 config CC3XX_BACKEND
 	bool
+	# default y if CRYPTOCELL_USABLE
```

Comment:
No commented code.

It is not clear to a reader if this is done intentionally or not.

Reply:
This intentional as we haven't added the new driver for CC3XX. The detail of CRYPTOCELL_USABLE is important for TF-M build

Reply:
This should be fixed everywhere in this patchset, not just on this line.

Reply:
The new driver will come as part of this PR, at which time all currently commented out configurations will be addressed